### PR TITLE
Saleor 3087 Paginate attribute values in filters

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it adds more attribute values fetching.

**PR intended to be tested with API branch:**  master

### Pull Request Checklist

1. [x] This code contains UI changes
2. [x] All visible strings are translated with proper context including data-formatting
3. [x] Attributes `[data-test-id]` are added for new elements
4. [x] Changes are mentioned in the changelog
5. [x] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3087 Display attribute values in filter list](https://app.clickup.com/t/2549495/SALEOR-3087)

**Review by eCzerniak — APPROVED** [review]

**Review by pwgryglak — APPROVED** [review]
looks ok

**Review by mmarkusik — CHANGES_REQUESTED** [review]

File: src/products/queries.ts [hunk 1]
```diff
@@ -76,32 +67,14 @@ const initialProductFilterAttributesQuery = gql`
...
-          }
         }
       }
     }
   }
 `;
 export const useInitialProductFilterAttributesQuery = makeQuery<
   InitialProductFilterAttributes,
-  InitialProductFilterAttributesVariables
+  never
```
[the remark this answers is not in the document]
Comment (orzechdev) [reply src/products/queries.ts:77]: Query doesn't have variables anymore.

File: src/components/Filter/FilterContent/FilterContentBody.tsx [hunk 2]
```diff
@@ -74,10 +74,6 @@ const FilterContentBody: React.FC<FilterContentBodyProps> = ({
   const intl = useIntl();
   const classes = useStyles({});
 
-  if (!filter?.active) {
```
[the remark this answers is not in the document]
Comment (orzechdev) [reply src/components/Filter/FilterContent/FilterContentBody.tsx:77]: No, it's old way of opening content body of the filter.

**Review by SektorDV — APPROVED** [review]

**Review by mmarkusik — APPROVED** [review]

**Pull request merged**